charts/kafka-operator/templates/_helpers.tpl

```diff
@@ -52,3 +52,17 @@ Compute operator prometheus metrics auth proxy service account
 {{ default "default" .Values.prometheusMetrics.authProxy.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Sidecar implementation details
+*/}}
+{{- define "chart.additionalSidecars"}}
+{{ toYaml .Values.additionalSidecars }}
```

Comment:
is is this define really required. would be easier to just inline the `{{ toYaml .Values.additionalSidecars }}` in the template directly?

Reply:
In fact, I take it back - having a define here would allow us to better control the templating in the dowstream charts defining this as a sub-chart.
helm defines are overridable in downstream charts

Reply:
Acknowledged.